ftag/track_selector.py

```diff
@@ -42,7 +42,7 @@ def __call__(self, tracks: np.ndarray) -> np.ndarray:
             if issubclass(tracks[var].dtype.type, np.floating):
                 tracks[var][rm_idx] = np.nan
             elif issubclass(tracks[var].dtype.type, np.integer):
-                tracks[var][rm_idx] = -1
+                tracks[var][rm_idx] = 0
```

Comment:
What's the motivation for using 0 over -1? -1 should look more like the padded tracks no?

Reply:
-1 doesn't work with unsigned types from H5 files.
For example `numberOfSCTSharedHits` has `unsigned char` type

Reply:
ah yeah good thinking :) ok and for signed either should be fine as we set the valid flag anyways. Ok I'll merge this!